runtime/testy/stateManager.py

```diff
@@ -143,7 +143,7 @@ def send_addr(self, process_name):
     self.processMapping[process_name].send(self.state["dawn_addr"][0])
 
   def student_upload(self):
-    #TODO Implement Student Upload Confirmation
+    self.badThingsQueue.put(BadThing(sys.exc_info(), None, BAD_EVENTS.ENTER_IDLE, False))
     self.processMapping[PROCESS_NAMES.TCP_PROCESS].send([ANSIBLE_COMMANDS.STUDENT_UPLOAD, True])
```

Comment:
Would there ever be a situation where you write to ansible with `ANSIBLE_COMMANDS.STUDENT_UPLOAD`, but set the data bit to `False` rather than `True`?